sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/ln/LnKey.java

```diff
@@ -33,5 +34,25 @@ public LnKey(TLN tln) {
         this.lnClass = tln.getLnClass().get(0);
         this.prefix = tln.getPrefix();
     }
+
+    public static DataAttributeRef updateDataRef(TAnyLN anyLN, DataAttributeRef dataAttributeRef) {
```

Comment:
What does this method is doing here ? How is it related to LnKey class ?
This should be in DataAttributeRef I think.

Reply:
Right

Reply:
`DataAttributeRef` is being replaced with another object `DoLinkedToDa`